database/migrations/2023_12_25_190920_delete_internet_access_role.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use App\Models\Role;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        if (Role::where('name', 'internet-user')->first() !== null) {
+            DB::table('roles')->delete(Role::where('name', 'internet-user')->first()->id);
+        }
```

Comment:
```suggestion
        if (Role::firstWhere('name', 'internet-user') !== null) {
            DB::table('roles')->delete(Role::firstWhere('name', 'internet-user')->id);
        }
```

Or maybe even `DB::table('roles')->delete(Role::firstWhere('name', 'internet-user')?->id);` without the first if. I don't know what delete with null as id does.